src/ufoLib2/objects/layerSet.py

```diff
@@ -86,30 +98,30 @@ def __attrs_post_init__(self):
                 )
         elif len(self._layers) == 1:
             # there's only one, we assume it's the default
-            self.defaultLayer = next(self._layers.values())
+            self.defaultLayer = self[next(iter(self._layers.keys()))]
         else:
             if DEFAULT_LAYER_NAME not in self._layers:
                 raise ValueError("default layer not specified")
-            self.defaultLayer = self._layers[DEFAULT_LAYER_NAME]
+            self.defaultLayer = self[DEFAULT_LAYER_NAME]
 
     @classmethod
-    def read(cls, reader, lazy=True):
-        layers = OrderedDict()
+    def read(cls, reader: UFOReader, lazy: bool = True) -> "LayerSet":
         """Instantiates a LayerSet object from a :class:`fontTools.ufoLib.UFOReader`.
 
         Args:
             path: The path to the UFO to load.
             lazy: If True, load glyphs, data files and images as they are accessed. If
                 False, load everything up front.
         """
+        layers: OrderedDict[str, Union[Layer, Placeholder]] = OrderedDict()
         defaultLayer = None
 
         defaultLayerName = reader.getDefaultLayerName()
 
         for layerName in reader.getLayerNames():
             isDefault = layerName == defaultLayerName
             if isDefault or not lazy:
-                layer = cls._loadLayer(reader, layerName, lazy, isDefault)
+                layer = cls._loadLayer(reader, layerName, lazy)
```

Comment:
Why is `isDefault` gone?

Reply:
It is only passed in `(_)loadLayer` when passed to a UFOWriter and we thought that makes no sense because it's an attribute used for writing.